decks.js

```diff
@@ -1,1194 +1,1293 @@
 const decks = [
-  {
-    name: "Animated Movies",
-    author: "N-Shar-ma",
-    instructions:
-      "The emoji/s on each card are a clue for an animated movie. You can flip (click) the card to see an additional hint. Drag and drop a card onto the matching text clue on the right, which will be the movie name in easy mode, and a character/place in the movie in the challenging mode. You can make a maximum of 3 wrong matches.",
-    cards: [
-      {
-        name: "Frozen",
-        content: {
-          primary: "❄️",
-          secondary: "🏔️🚪",
+    {
+      name: "Animated Movies",
+      author: "N-Shar-ma",
+      instructions:
+        "The emoji/s on each card are a clue for an animated movie. You can flip (click) the card to see an additional hint. Drag and drop a card onto the matching text clue on the right, which will be the movie name in easy mode, and a character/place in the movie in the challenging mode. You can make a maximum of 3 wrong matches.",
+      cards: [
+        {
+          name: "Frozen",
+          content: {
+            primary: "❄️",
+            secondary: "🏔️🚪",
+          },
+          hint: "Has 2 female protagonists",
+          matchValue: "Arendelle",
+          seenHint: false,
```

Comment:
Please note that you modified the formatting across the whole file. Can you please limit the changes to only adding new deck?